react/Utils.ts

```diff
@@ -45,4 +60,23 @@ export const validateAddress = (
   }
 }
 
-export default { validateAddress }
+export const createEmptyAddress = (disposable = false): Address => {
```

Comment:
Why isn't `isDisposable = null` here?

Reply:
I don't wan't to `isDisposable` to be nullable, although it wouldn't make a difference here because both values are falsy 😆

Reply:
But at the same time, you don't want to make it a required field and generate a breaking change, right? Btw making it non-nullable makes more sense to me as well

Reply:
> But at the same time, you don't want to make it a required field and generate a breaking change, right?

why would it be a breaking change if this is a new function? 🤔

Reply:
I mean, making the `isDisposable` a required field in the `Address` type, not the new function

Reply:
oh, I don't plan to do that, I think the `isDisposable` is optional (imagine the addresses that were created before the introduction of this feature)

Reply:
Mas e se ao invés de passar `null`, o dev passar `{}`? Vai gerar uma inconsistência entre a função e um "empty address de verdade". Ou você acha que não?

Reply:
ahh, teoricamente a pessoa nao deveria poder fazer isso por conta dos tipos do typescript